docs/DeveloperGuide.md

```diff
@@ -237,70 +237,403 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* *`    | Product Manager                          | see a calendar view of my meetings                                          | know my available slots to schedule new meetings |
+| `* *`    | Product Manager                          | toggle between daily, weekly and monthly view                               | have both a macro and micro view of my schedule |
+| **Communication** |
+| `* * *`  | Product Manager                          | distinguish between the developers and the stakeholders in a project easily | remember the different roles       |
+| `*`      | Product Manager                          | view contacts of people involved in a product                               | communicate with relevant people                          |
+| `*`      | Product Manager                          | check the product’s completion stage                                        | quickly communicate any needs to the Dev Team and/or stakeholders |
+
+
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
 
-*{More to be added}*
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Productiv` and the **Actor** is the `user`, unless specified otherwise)
+
+#### Mode
+
+**Use case: `UC01 - Switch Mode`**
+
+**MSS**
+
+1. User chooses to switch the mode.
+2. User enters the command to switch mode into the input box.
+3. Productiv switches to the expected mode.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. Produtiv detects an error in the command.
+     
+     * 2a1. Productiv displays an error message
```

Comment:
In here and elsewhere, for consistency, **can we ensure all the lines end with periods (.)?**

Reply:
Have seen that the rest have periods now. How about this line too?